src/renderer/api/kube-watch-api.ts

```diff
@@ -57,16 +56,13 @@ export class KubeWatchApi {
   }
 
   @computed get apis(): string[] {
-    const { cluster, namespaceStore } = this;
-    const activeApis = Array.from(this.subscribers.keys());
-
-    return activeApis.map(api => {
-      if (!cluster.isAllowedResource(api.kind)) {
+    return Array.from(this.subscribers.keys()).map(api => {
+      if (!this.getCluster?.().isAllowedResource(api.kind)) {
         return [];
       }
 
       if (api.isNamespaced) {
-        return namespaceStore.getContextNamespaces().map(namespace => api.getWatchUrl(namespace));
+        return this.getNamespaces().map(namespace => api.getWatchUrl(namespace));
       } else {
```

Comment:
If the getter isn't defined we should either throw a defined error or return `[]` instead of having the error be "cannot call undefined"

Reply:
Maybe, but it's used only internally and not critical imho.